packages/mermaid/src/docs/syntax/sequenceDiagram.md

```diff
@@ -369,6 +369,30 @@ sequenceDiagram
     API-->BillingService: Start billing process
 ```
 
+## Custom Grouping
```

Comment:
Make sure [MERMAID_RELEASE_VERSION](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/community/development.md#3-update-documentation) is used for all new features.

Reply:
Added